src/test/ui/or-patterns/or-patterns-syntactic-pass.rs

```diff
@@ -21,28 +21,28 @@ accept_pat!([p | q]);
 #[cfg(FALSE)]
 fn or_patterns() {
     // Top level of `let`:
-    let (| A | B);
+    let (A | B);
```

Comment:
Just looked at this change list again since it was linked and these slipped in (presumably from fmt-on-save) and shouldn't've.